src/Data/Nat/Properties.agda

```diff
@@ -286,6 +286,11 @@ n≤0⇒n≡0 z≤n = refl
 ≤∧≢⇒< {_} {suc n} (s≤s m≤n) 1+m≢1+n =
   s≤s (≤∧≢⇒< m≤n (1+m≢1+n ∘ cong suc))
 
+≤∧≮⇒≡ : ∀ {m n} → m ≤ n → m ≮ n → m ≡ n
+≤∧≮⇒≡ {zero} {zero} z≤n m≮n = refl
+≤∧≮⇒≡ {zero} {suc n} z≤n m≮n = contradiction (s≤s z≤n) m≮n
+≤∧≮⇒≡ {suc m} {suc n} (s≤s m≤n) m≮n = cong suc (≤∧≮⇒≡ m≤n (m≮n ∘ s≤s))
```

Comment:
Please can you align the function arguments as per the [style guide](https://github.com/agda/agda-stdlib/blob/master/notes/style-guide.md#function-arguments) as it tends to make proofs easier to read. At some point someone is going to write a linter for Agda and everyone will be happy :smile:

Reply:
Why not use `≤-antisym` in combination with `≮⇒≥`?